### PR TITLE
Sec Hud Nerf

### LINKED ISSUE #8517: Permawanted after getting plastic surgery
I was set to arrest and got plastic surgery done to me, after the plastic surgery was done not only was I set to arrest on sec HUD, I was completely unable to remove that status and even kept it when unknown. Please fix, Sec HUDs or security consoles couldn't set me off arrest.

### LINKED ISSUE #8654: Wanted status persists even if you get a gas mask/new ID
1. Get set to arrest
2. Dump your old ID
3. Put on new ID/gas mask
4. Still have a big W above your head, get arrested

### PR DESCRIPTION
**Pull request by astralenigma**
Just joking this only fixes the permawanted status.
Fixes #8654 
Fixes #8517

### CODE REVIEW

**Comment by PKPenguin321** [comment]
nice

**Comment by KorPhaeron** [comment]
Finally, thank you.

**Comment by Supermichael777** [comment]
squash those comits

File: code/modules/surgery/plastic_surgery.dm [hunk 1]
```diff
@@ -16,9 +16,15 @@
...
+			var/mob/living/carbon/human/H = target
+			H.sec_hud_set_ID()
 	else
 		var/oldname = target.real_name
 		target.real_name = target.dna.species.random_name(target.gender,1)
 		var/newname = target.real_name	//something about how the code handles names required that I use this instead of target.real_name
 		user.visible_message("[user] alters [oldname]'s appearance completely, they are now [newname]!", "<span class='notice'>You alter [oldname]'s appearance completely, they are now [newname].</span>")
+		if(ishuman(target))
+			var/mob/living/carbon/human/H = target
+			H.sec_hud_set_ID()
```
Comment (phil235) [inline code/modules/surgery/plastic_surgery.dm]: you can remove the duplicate code by simply having the three lines once just before the return 1

**Comment by Dorsisdwarf** [comment]
@astralenigma 

Oi. conflicts

**Comment by phil235** [comment]
I made a new version of this PR.

**Pull request closed without merging**